lib/matplotlib/tests/test_basic.py

```diff
@@ -3,10 +3,17 @@
 
 import six
 
+import warnings
+
 from nose.tools import assert_equal
 
+from matplotlib.cbook import MatplotlibDeprecationWarning
 from matplotlib.testing.decorators import knownfailureif
-from pylab import *
+with warnings.catch_warnings():
+    warnings.filterwarnings('ignore',
+                            'The finance module has been deprecated in mpl 2',
+                            MatplotlibDeprecationWarning)
+    from pylab import *
```

Comment:
Well, that's a question for a different issue/PR, I think.

Anything done here will cause a conflict with `master`, probably. We could backport your entire change to this file, but I thought that might be a bit drastic.